### ISSUE TITLE
Incorrect information

### ISSUE THREAD

**Issue by siddudubey** [MicrosoftDocs/sql-docs#4082]
The page says "The SQL Server Browser does not return port information for the default instance."; however, I just captured the UDP packets coming from 1434 and verified that the Browser returns the port number for default instance as well. I am using SQL Server 2016.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 98647be2-f5c6-2c96-d1f3-aab8a847425b
* Version Independent ID: 139c1aaf-72e2-9a90-ec2f-0e46f5cbc36a
* Content: [SQL Server Browser Service (Database Engine and SSAS) - SQL Server](https://docs.microsoft.com/en-us/sql/database-engine/configure-windows/sql-server-browser-service-database-engine-and-ssas?view=sql-server-ver15#feedback)
* Content Source: [docs/database-engine/configure-windows/sql-server-browser-service-database-engine-and-ssas.md](https://github.com/MicrosoftDocs/sql-docs/blob/live/docs/database-engine/configure-windows/sql-server-browser-service-database-engine-and-ssas.md)
* Product: **sql**
* Technology: **configuration**
* GitHub Login: @MikeRayMSFT
* Microsoft Alias: **mikeray**

**Comment by WilliamAntonRohm** [comment]
@siddudubey -- thank you for your feedback and your research.

@MikeRayMSFT -- please look into this issue #4082.

**Comment by siddudubey** [comment]
Just to add more details, my client is written in Java and it uses the  jTDS driver (http://jtds.sourceforge.net), I went through the source code of the driver to find out why am I getting default instance port number from the Browser, and now I know.  jTDS driver sends UDP message 0x02 to the browser service, which is CLNT_BCAST_EX message of SQL Server Resolution Protocol (see here https://docs.microsoft.com/en-us/openspecs/windows_protocols/mc-sqlr/a3035afa-c268-4699-b8fd-4f351e5c8e9e). The Browser service sends details of ALL instances (including default instance) when it gets CLNT_BCAST_EX or CLNT_UCAST_EX message. So yes, the line I have highlighted is incorrect. It may be correct when the client is using the drivers shipped by Microsoft (ODBC, native driver etc) because they probably use CLNT_UCAST_INST message which returns details of the requested instance only.
